src/styles/globals.css

```
@@ -0,0 +1,35 @@
+  width: 100%;
+  max-width: 430px;    
+  margin: 0 auto;      
+  min-height: 100vh;
+  min-height: 100dvh;
+  background: var(--color-white);
+  padding-left: max(16px, env(safe-area-inset-left));
+  padding-right: max(16px, env(safe-area-inset-right));
+  padding-bottom: max(16px, env(safe-area-inset-bottom));
+}
+
+@import "../design-system/typography.css";
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)